vita3k/interface.cpp

```diff
@@ -272,6 +273,22 @@ static ExitCode load_app_impl(Ptr<const void> &entry_point, HostState &host, Gui
 
     host.renderer->features.hardware_flip = host.cfg.hardware_flip;
 
+    // Load user driver
```

Comment:
Use an array similar to pre-loaded libraries in order to add more.

extract reading file and loading to a common function to this and pre-loaded

Reply:
is not possible, need use diferent things, no same partition/folder need obligatory using totaly diferent code